style.css

```diff
@@ -378,4 +378,91 @@ body
     {
         padding: 40px 0;
     }
+}
+
+/* Seventh section solutions */
+
+.solution
+{
+    width: 100%;
+    padding: 120px 0;
+}
+
+.solution h2
+{
+    font-size: 42px;
+    font-weight: 500;
+    text-align: center;
+    margin-bottom: 80px;
+}
+
+.solution .solution-contents
+{
+    display: flex;
+    justify-content: space-between;
+    gap: 20px;
+
+}
+
+.solution-contents .solution-image
+{
+    width: 25%;
+    height: 600px;
+    max-width: 600px;
+    background: linear-gradient(0deg, #0009 30%, #fff0);
+    position: relative;
+
+}
+
+.solution-contents .solution-image img
+{
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    position: relative;
+    z-index: -1;
+}
+
+.solution-contents .solution-image h3
+{
+    width: 100%;
+    position: absolute;
+    top: 85%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    font-size: 28px;
+    color: #fff;
+    text-align: center;
+}
+
+@media (max-width: 800px)
+{
+    .solution
+    {
+        padding: 80px 0;
+    }
+
+    .solution h2
+    {
+        font-size: 34px;
+        margin-bottom: 44px;
+    }
+
```

Comment:
consider the font-weight in mobile view, i think you need to revisit that in all sections.